### PR TITLE
Add startAfter argument to start scrolling after a duration

### PR DESCRIPTION
**Pull request by hacker1024**
As discussed in #39.

### CODE REVIEW

File: lib/marquee.dart [hunk 1]
```diff
@@ -268,6 +273,20 @@ class Marquee extends StatefulWidget {
   /// * [scrollAxis] to change the axis in which the scrolling takes place.
   final double velocity;
 
+  /// Start scrolling after this duration after the widget is first displayed.
+  ///
+  /// ## Sample code
+  ///
+  /// This Marqee starts scrolling one second after being displayed.
```
Comment (marcelgarus) [inline lib/marquee.dart:280]:
```suggestion
  /// This [Marquee] starts scrolling one second after being displayed.
```

**Review by marcelgarus — CHANGES_REQUESTED** [review]

**Comment by marcelgarus** [comment]
Apart from the typo, this looks really good to me. Looking forward to merging this! 😄
Also, thanks for taking the time to contribute and creating a PR, I really appreciate that.

**Comment by hacker1024** [comment]
Fixed.

> Also, thanks for taking the time to contribute and creating a PR, I really appreciate that.

No problem, it was surprisingly simple.

**Comment by marcelgarus** [comment]
Nice, I published this as version 1.6.0 on pub.dev

**Pull request merged**